packages/components/tests/e2e/specs/vsCodeExtension.spec.js

```diff
@@ -524,6 +524,35 @@ context('VS Code Extension', () => {
         .invoke('text')
         .should('not.match', /SELECT.*FROM/);
     });
+
+    it('applies filters properly', () => {
+      const listClassFor = (objType) => `.details-search__block--${objType} li`;
+
+      cy.get(listClassFor('route')).should('have.length', 3);
+      cy.get(listClassFor('labels')).should('have.length', 4);
+      cy.get(listClassFor('package')).should('have.length', 4);
+      cy.get(listClassFor('class')).should('have.length', 11);
+      cy.get(listClassFor('function')).should('have.length', 20);
+      cy.get(listClassFor('query')).should('have.length', 32);
+
+      cy.get('.popper__button').click();
+
+      cy.get('#unlabeled-events').click();
+      cy.get(listClassFor('labels')).should('have.length', 4);
+      cy.get(listClassFor('function')).should('have.length', 6);
+      cy.get('#unlabeled-events').click();
+    });
+
+    it('shows all root events with disabled filter', () => {
+      cy.get('.tabs .tab-btn').last().click();
+
+      cy.get('.trace-node').should('have.length', 4);
+
+      cy.get('.popper__button').click();
+      cy.get('#limit-root-events').click();
+
+      cy.get('.trace-node').should('have.length', 12);
+    });
```

Comment:
Please add a test for the media request filter.

Reply:
I've added new HTTP server request event to `scenario.json` and added new test case for media requests filter. https://github.com/applandinc/appmap-js/pull/217/commits/82134753fbbc8ed84c34ccf49e283e08dc3dc948